scripts/api-extractor.ts

```diff
@@ -532,6 +532,15 @@ async function buildDocs({
         output.appendNode(pluginsTable);
       }
     }
+
+    private _getFilenameForApiItem(apiItem: ApiItem): string {
```

Comment:
Ain't this entire file a thing of beauty 😗 🤌

Reply:
Looks like the issue has now been fixed in api-documenter, so we should be able to skip this change if we update to [v7.13.68](https://github.com/microsoft/rushstack/blob/master/apps/api-documenter/CHANGELOG.md#71368).

Reply:
Bumped api-documenter in https://github.com/backstage/backstage/pull/8023.

Reply:
Pulled https://github.com/backstage/backstage/pull/7942/commits/6656e35e69a204761c7ae944636e8909cc29232c out of this PR :)

Reply:
Worked, and #8023 is merged :tada: